README.md

```diff
@@ -1,13 +1,3 @@
-# Project Happy Thoughts API
-
-Replace this readme with your own information about your project.
-
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
-
```

Comment:
You could add some text about the project, the purpose for example so if someone is viewing your project they can quickly get a sense of what it's about.